src/tailwind.js

```diff
@@ -13,7 +13,10 @@ const localConfig = {
       "wave-lines-light": "url('/static/images/common/wave-lines-light.png')",
     },
     fontFamily: {
-      roboto: ["Roboto", "sans-serif"],
+      "gilroy-light": ["Gilroy-Light", "sans-serif"],
+      "gilroy-medium": ["Gilroy-Medium", "sans-serif"],
+      "gilroy-bold": ["Gilroy-Bold", "sans-serif"],
+      "gilroy-extrabold": ["Gilroy-ExtraBold", "sans-serif"],
       ubuntu: ["Ubuntu", "sans-serif"],
     },
```

Comment:
nitpick (non-blocking): these are inherited from [ui components](https://github.com/TradeTrust/tradetrust-ui-components/blob/master/src/tailwind.js#L14). if im not wrong, will work without

Reply:
ok will update it to be like the one in creator.